astroquery/vizier/tests/test_vizier_remote.py

```diff
@@ -122,8 +124,9 @@ def test_multicoord(self):
 
     def test_findcatalog_maxcatalog(self):
         V = vizier.core.Vizier()
-        cats = V.find_catalogs('eclipsing binary', max_catalogs=5000)
-        assert len(cats) >= 468
+        with pytest.raises(TableParseError, match=r"Failed to parse VIZIER result"):
```

Comment:
do we expect this error? cc @keflavich 

(Same for L140)

Reply:
No, but this looks like an upstream error.  I'll post an issue about it.

Reply:
https://github.com/astropy/astroquery/issues/2537

Reply:
I am removing this `with pytest.raises` here and leaving it to the issue to resolve.